proto/one/common/configuration/configuration_note.proto

```diff
@@ -0,0 +1,30 @@
+syntax = "proto3";
+
+import "record_auditinfo.proto";
+import "jsonTicksDateTime.proto";
+
+option csharp_namespace ="ONE.Models.CSharp";
+
+message ConfigurationNote {
+    // guid identifier of the note
+    string id = 1;
+
+    // guid identifier of the configuration associated with the note
+    string configurationId = 2;
+
+    // time associated with the note
+    JsonTicksDateTime noteTime = 3;
+
+    // text of the note
+    string note = 4;
+    
+    // tags associated with this note
+    repeated string tags = 5;
```

Comment:
I think we need a separate file for tags to accomplish https://aquaticinformatics.atlassian.net/wiki/spaces/OP/pages/59080751/Method+Claros.Common.Configuration.API+Notes+Tags+GET

Reply:
Was planning to implement that endpoint using the existing KeyValue protobuf model where the key would be the configurationId and the value would be the space delimited list of tags.